pkg/scheduler/algorithm/predicates/csi_volume_predicate.go

```diff
@@ -19,9 +19,11 @@ package predicates
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
please remove `v1`

Reply:
This is autoadded by goimports. It's not worth fighting the tool, in my opinion

Reply:
Yes, but it's not worth introducing a dummy alias. I'm now turning to `gofmt`, and I have a long post at https://groups.google.com/d/msg/kubernetes-dev/Y2ShVRrU4xM/LKfaiFqAAwAJ explaining how to configure in vscode.

Reply:
I agree that it's annoying, but it changes nothing else in the code. I saw it in this PR and decided not to mention it since that's just extra work for the author and us the reviewers.

Incidentally I recall sending feedback on this behavior and getting a "working as intended" response, but I couldn't find that response when I looked for it. I think the tool assumes that our module is actually named "core" and uses semver so that major versions have different import paths. Therefore, the `v1` alias is an indication that no, actually our module is named "v1".

I know it's convention not to alias this import, but I think if we started aliasing it as `core` that things like `core.Pod` would make more sense.

Reply:
https://github.com/golang/go/issues/30051